### DISCUSSION TITLE
Welcome to Incognito-tab-mover Discussions!

### DISCUSSION THREAD

**Discussion by rohit-wadhwa** [rohit-wadhwa/Incognito-tab-mover#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by omrylivny** [comment]
Right now, it seems like incognito tabs can only be moved one by one, which can be tedious if you have a few incognito tabs open. Adding an option to move bulk group of tabs would be a great improvment which will make the process much efficient for users managing multiple tabs. Please consider and thanks for the helpful extention.

**Reply by rohit-wadhwa** [reply]
Sure, I will work on it and update this.

**Comment by omrylivny** [comment]
Thanks a lot.

**Reply by rohit-wadhwa** [reply]
@omrylivny , here you go: https://github.com/rohit-wadhwa/Incognito-tab-mover/discussions/3

Also, please star the repo and, after testing, add a review on the Chrome Store. thanks.

**Comment by omrylivny** [comment]
You're amazing and it works great.
Appreciate the quick response.

Best,
Omry.

**Reply by rohit-wadhwa** [reply]
Your Welcome
